### PR TITLE
Update Lyo Core to use Jax-rs 2.0, and be independent of Wink (as well as any other implementation)

### PR DESCRIPTION
**Pull request by jadelkhoury**
Update Lyo Core to use Jax-rs 2.0, and be independent of Wink (as well as any other implementation)

### CODE REVIEW

File: org.eclipse.lyo.oslc4j.build/pom.xml [hunk 1]
```diff
@@ -37,7 +37,7 @@
     <module>../org.eclipse.lyo.oslc4j.provider.jena</module>
     <module>../org.eclipse.lyo.oslc4j.provider.json4j</module>
     <module>../org.eclipse.lyo.core.utils</module>
-    <module>../org.eclipse.lyo.oslc4j.wink</module>
+    <!-- <module>../org.eclipse.lyo.oslc4j.wink</module> -->
```
Comment (berezovskyi) [inline org.eclipse.lyo.oslc4j.build/pom.xml]: Breaking change, requires a major version increase to 3.1.0. CC @jamsden

File: org.eclipse.lyo.oslc4j.core/pom.xml [hunk 2]
```diff
@@ -21,26 +21,52 @@
...
+		      <version.jersey>2.27</version.jersey>
     </properties>
     <!-- Disable doclint for Java 1.8 (for now) -->
     <dependencies>
         <dependency>
             <groupId>javax.ws.rs</groupId>
-            <artifactId>jsr311-api</artifactId>
-            <version>1.1.1</version>
+            <artifactId>javax.ws.rs-api</artifactId>
+            <version>2.1</version>
```
Comment (berezovskyi) [inline org.eclipse.lyo.oslc4j.core/pom.xml]: Requires CQ

File: org.eclipse.lyo.oslc4j.core/pom.xml [hunk 3]
```diff
@@ -21,26 +21,52 @@
...
+           <type>pom</type>
         </dependency>
         <dependency>
-            <groupId>org.apache.jena</groupId>
-            <artifactId>apache-jena-libs</artifactId>
-            <type>pom</type>
-         </dependency>
+            <groupId>org.glassfish.jersey.containers</groupId>
+            <artifactId>jersey-container-servlet</artifactId>
+            <version>${version.jersey}</version>
```
Comment (berezovskyi) [inline org.eclipse.lyo.oslc4j.core/pom.xml]: v2.27 requires CQ

File: org.eclipse.lyo.oslc4j.core/pom.xml [hunk 4]
```diff
@@ -21,26 +21,52 @@
...
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.glassfish.jersey.core</groupId>
+            <artifactId>jersey-server</artifactId>
+            <version>${version.jersey}</version>
```
Comment (berezovskyi) [inline org.eclipse.lyo.oslc4j.core/pom.xml]: v2.27 requires CQ

File: org.eclipse.lyo.oslc4j.core/pom.xml [hunk 5]
```diff
@@ -21,26 +21,52 @@
...
+             <scope>test</scope>
+        </dependency>
+          <dependency>
+            <groupId>org.glassfish.jersey.core</groupId>
+            <artifactId>jersey-common</artifactId>
+            <version>${version.jersey}</version>
```
Comment (berezovskyi) [inline org.eclipse.lyo.oslc4j.core/pom.xml]: v2.27 requires CQ

File: org.eclipse.lyo.oslc4j.core/pom.xml [hunk 6]
```diff
@@ -21,26 +21,52 @@
...
+             <scope>test</scope>
+        </dependency>
         <dependency>
+        <groupId>org.glassfish.jersey.inject</groupId>
+		<artifactId>jersey-hk2</artifactId>
+		<version>${version.jersey}</version>
+		<!-- FIXME: jersey-hk2 is only needed for testing, but the build fails if scope is set to test. -->
```
Comment (berezovskyi) [inline org.eclipse.lyo.oslc4j.core/pom.xml]: FIXME sanity check

**Comment by jadelkhoury** [comment]
We abandon this pull request, which is superseded by https://github.com/eclipse/lyo.core/pull/63

**Pull request closed without merging**